hedera-node/src/main/java/com/hedera/services/grpc/marshalling/ImpliedTransfers.java

```diff
@@ -41,38 +45,54 @@
 public class ImpliedTransfers {
 	private final ImpliedTransfersMeta meta;
 	private final List<BalanceChange> changes;
+	private final List<Pair<EntityId, List<CustomFee>>> entityCustomFees;
```

Comment:
Something like `involvedTokenFeeSchedules` might be more readable here.

Reply:
Fixed !